presto-tpcds/src/main/java/io/prestosql/plugin/tpcds/TpcdsMetadata.java

```diff
@@ -73,6 +73,12 @@ public TpcdsMetadata()
         this.tableNames = tableNames.build();
     }
 
+    @Override
+    public boolean schemaExists(ConnectorSession session, String schemaName)
+    {
+        return schemaNameToScaleFactor(schemaName) < 0 ? false : true;
```

Comment:
- sf0 does not exists
- this needs to be handled the same way as in `io.prestosql.plugin.tpcds.TpcdsMetadata#getTableHandle`

```suggestion
        return schemaNameToScaleFactor(schemaName) > 0;
```